doc-content/enterprise-only/admin-and-config/logging-events-database-proc.adoc

```diff
@@ -0,0 +1,78 @@
+[id='logging-events-database-proc']
+= Logging events to database
+
+You can log an event to a database that occurs on runtime in a process instance, node instance or variable instance.
+
+.Procedure
+. Map log classes to the data source that enables the data source to accept the log entries.
+* On {EAP}, edit the data source properties in the `persistence.xml` file.
+[source,code]
++
+----
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<persistence  version="1.0"  xsi:schemaLocation=
+    "http://java.sun.com/xml/ns/persistence
+     http://java.sun.com/xml/ns/persistence/persistence_1_0.xsd
+     http://java.sun.com/xml/ns/persistence/orm
+     http://java.sun.com/xml/ns/persistence/orm_1_0.xsd"
+  xmlns:orm="http://java.sun.com/xml/ns/persistence/orm"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://java.sun.com/xml/ns/persistence">
+
+  <persistence-unit name="org.jbpm.persistence.jpa">
+    <provider>org.hibernate.ejb.HibernatePersistence</provider>
```

Comment:
We may rather use `org.hibernate.jpa.HibernatePersistenceProvider`

Reply:
+1